src/preset.ts

```diff
@@ -1649,6 +2072,39 @@ export function testArgs(preset: TestPreset): string[] {
     return result;
 }
 
+export function packageArgs(preset: PackagePreset): string[] {
+    const result: string[] = [];
+
+    // -C semicolon;separated;list;of;configurations;to;pack
+    const configurations: string | undefined = preset.configurations?.join(";");
+    configurations && result.push(`-C ${configurations}`); // should this be 2 args or 1 with space in between -C and configurations list?
+    // -G semicolon;separated;list;of;generators;used
+    const generators: string | undefined = preset.generators?.join(";");
+    generators && result.push(`-G ${generators}`); // should this be 2 args or 1 with space in between -G and generators list?
+
+    // cpack variables: -D var=val
+    if (preset.variables) {
+        util.objectPairs(preset.variables).forEach(([key, value]) => {
+            result.push(`-D ${key}=${value}`);
+        });
+    }
+
+    preset.configFile && result.push('--config', preset.configFile);
+    preset.packageName && result.push('-P', preset.packageName);
+    preset.packageVersion && result.push('-R', preset.packageVersion);
+    preset.packageDirectory && result.push('-B', preset.packageDirectory);
+
+    // Output
+    if (preset.output) {
+        preset.output.verbose && result.push('-V');
+        preset.output.debug && result.push('--debug');
+    }
+
+    return result;
+}
+
+// TODO: If needed, implement workflowArgs (or workflowStepsArgs) to return an array of args for each contained preset.
```

Comment:
Stale comment?

Reply:
Yes but there is an idea behind worth discussing. All other presets can offer their command lines. Since the workflow is a collection of presets, we can offer a list of commands for it as well. I removed the comment and will open an issue on GitHub.